bootstrap.json

```diff
@@ -0,0 +1,9 @@
+{
+    "deps": {
+        "github.com/golang/dep/cmd/dep": "v0.4.1",
+        "github.com/golang/lint/golint": "master",
```

Comment:
Any reason why we don't use `dep` to pin/install the other project from this file ?

Reply:
just my opinion here, but I'm thinking we could leave these non-critical deps unpinned, in which case I don't think using `dep` to pull them makes sense

Reply:
The reasons I didn't use dep for these are:
 - they were previously unpinned and being `go get`
 - I'm not sure it makes sense to vendor them as they're not used by the project's code, but rather for project management.

One thing we should do either way, or at least contemplate (perhaps add a switch) is the event where the user already has these tools available on his system. We might not want to override what he's already using.

Reply:
+1 for not vendoring them. But I this we should pin them since they're used in our CI and could prevent us from building if they start failing.

Reply:
I don't have a strong opinion on pinning `golint` et al., but I'm fine with leaving them unpinned. Worst case scenario we'll see the CI fail and it'll force us to pin them to a known working version for the tag we want to build... On the bright side It'll force us to keep our code up-to-date, and there's no chance that leaving them unpinned will affect the final binaries that are built (unlike `dep`).